From 65b20eedbc431e192d57a602feb625811a0fa8b1 Mon Sep 17 00:00:00 2001
Message-Id: <65b20eedbc431e192d57a602feb625811a0fa8b1.1442506740.git.erack@redhat.com>
From: Eike Rathke <erack@redhat.com>
Date: Thu, 17 Sep 2015 17:12:03 +0200
Subject: [PATCH] Resolves: tdf#85979 TextToColumns is overwriting, so set at
 ScImportExport
MIME-Version: 1.0
Content-Type: multipart/mixed; boundary="------------erAck-patch-parts"

This is a multi-part message in MIME format.
--------------erAck-patch-parts
Content-Type: text/plain; charset=UTF-8; format=fixed
Content-Transfer-Encoding: 8bit


Change-Id: Id0fec512b53aab47f9951056e8726af0cce45708
(cherry picked from commit 624ab2e5a52e9b8aa139bae4cd1436d23b80bf8c)
---
 sc/source/ui/view/cellsh2.cxx | 1 +
 1 file changed, 1 insertion(+)


--------------erAck-patch-parts
Content-Type: text/x-patch; name="0001-Resolves-tdf-85979-TextToColumns-is-overwriting-so-s.patch"
Content-Transfer-Encoding: 8bit
Content-Disposition: attachment; filename="0001-Resolves-tdf-85979-TextToColumns-is-overwriting-so-s.patch"

diff --git a/sc/source/ui/view/cellsh2.cxx b/sc/source/ui/view/cellsh2.cxx
index 7b7189c..2b80c37 100644
--- a/sc/source/ui/view/cellsh2.cxx
+++ b/sc/source/ui/view/cellsh2.cxx
@@ -977,6 +977,7 @@ void ScCellShell::ExecuteDB( SfxRequest& rReq )
                         aImport.SetExtOptions( aOptions );
                         aImport.SetApi( false );
                         aImport.SetImportBroadcast( true );
+                        aImport.SetOverwriting( true );
                         aStream.Seek( 0 );
                         aImport.ImportStream( aStream, OUString(), FORMAT_STRING );
 

--------------erAck-patch-parts--


